label_studio/data_manager/managers.py

```diff
@@ -436,7 +437,8 @@ def apply_filters(queryset, filters, project, request):
             cast_value(_filter)
             filter_expressions.append(Q(**{field_name: _filter.value}))
 
-    logger.debug(f'Apply filter: {filter_expressions}')
+    # Do not uncomment this. Stringifying filter_expressions evaluates the (sub)queryset.
```

Comment:
can we just delete this?  Why are we leaving this logging statement in here?

Reply:
I wanted to leave an explanation in code as to why this was remove and shouldn't be re-added

Reply:
hmm.. normally I would say remove it but in this case its a bit of a warning, so yeah there's value in leaving it in there

Reply:
I would maybe make the comment along the lines of:

```
"""WARNING: Stringifying filter_expressions will evaluate the (sub)queryset.
    Do not use a log in the following manner:
    logger.debug(f'Apply filter: {filter_expressions}')
"""
```